modules/cluster-logging-exported-fields-top-level-fields.adoc

```diff
@@ -10,38 +9,52 @@
 
 The top level fields may be present in every record.
 
-=== message
-
+== message
 
 Original log entry text, UTF-8 encoded This field may be absent or empty if a non-empty `structured` field is present. See the description of `structured` for more.
 
-=== structured
+[horizontal]
+Data type:: text
+Example value:: `HAPPY`
 
+== structured
 
 Original log entry as a structured object. This field may be present if the forwarder was configured to parse structured JSON logs. If the original log entry was a valid structured log, this field will contain an equivalent JSON structure. Otherwise this field will be empty or absent, and the `message` field will contain the original log message. The `structured` field has whatever sub-fields are included in the log message, there are no restrictions defined here.
 
-=== @timestamp
+[horizontal]
+Data type:: group
+Example value:: map[message:starting fluentd worker pid=21631 ppid=21618 worker=0 pid:21631 ppid:21618 worker:0]
 
+== @timestamp
 
 A UTC value that marks when the log payload was created or, if the creation time is not known, when the log payload was first collected. The “@” prefix denotes a field that is reserved for a particular use. By default, most tools look for “@timestamp” with ElasticSearch.
 
-=== hostname
+[horizontal]
+Data type:: date
+Example value:: `2015-01-24 14:06:05.071000000 Z`
 
+== hostname
 
 The name of the host where this log message originated. In a Kubernetes cluster, this is the same as `kubernetes.host`.
 
-=== ipaddr4
+[horizontal]
+Data type:: keyword
 
+== ipaddr4
 
 The IPv4 address of the source server. Can be an array.
 
-=== ipaddr6
+[horizontal]
+Data type:: ip
 
+== ipaddr6
 
 The IPv6 address of the source server, if available. Can be an array.
 
-=== level
+[horizontal]
+Data type:: ip
 
+== level
 
 The logging level from various sources, including `rsyslog(severitytext property)`, python's logging module, and others.
```

Comment:
Should be "a Python logging module"? Capitalization and also inanimate nouns shouldn't have possessive forms.

Also, the list below (lines 63-68 I can't comment on) doesn't contain articles in front of items, but maybe it was intentional.

Reply:
Updated. Thanks. The list below comes from another canonical source. I agree with you, but don't want to tinker with a set of standard definitions for now.